packages/now-cli/src/commands/domains/add.ts

```diff
@@ -47,105 +49,119 @@ export default async function add(
     return 1;
   }
 
-  if (subdomain) {
-    output.error(
-      `You are adding '${domainName}' as a domain name containing a subdomain part '${subdomain}'\n` +
-        `  This feature is deprecated, please add just the root domain: ${chalk.cyan(
-          `${getCommandName(`domain add ${domain}`)}`
-        )}`
-    );
-    return 1;
-  }
-
   const addStamp = stamp();
-  const addedDomain = await addDomain(client, domainName, contextName);
 
-  if (addedDomain instanceof ERRORS.InvalidDomain) {
-    output.error(
-      `The provided domain name "${addedDomain.meta.domain}" is invalid`
-    );
-    return 1;
-  }
+  let aliasTarget = await addDomainToProject(client, projectName, domainName);
+
+  if (aliasTarget instanceof Error) {
+    if (
+      aliasTarget instanceof ERRORS.APIError &&
+      aliasTarget.code === 'ALIAS_DOMAIN_EXIST' &&
+      aliasTarget.project &&
+      aliasTarget.project.id
+    ) {
+      if (force) {
+        const removeResponse = await removeDomainFromProject(
+          client,
+          aliasTarget.project.id,
+          domainName
+        );
+
+        if (removeResponse instanceof Error) {
+          output.error(removeResponse.message);
+          return 1;
+        }
+
+        aliasTarget = await addDomainToProject(client, projectName, domainName);
+      }
+    }
 
-  if (addedDomain instanceof ERRORS.DomainAlreadyExists) {
-    output.error(
-      `The domain ${chalk.underline(
-        addedDomain.meta.domain
-      )} is already registered by a different account.\n` +
-        `  If this seems like a mistake, please contact us at support@vercel.com`
-    );
-    return 1;
+    if (aliasTarget instanceof Error) {
+      output.error(aliasTarget.message);
+      return 1;
+    }
   }
 
   // We can cast the information because we've just added the domain and it should be there
   console.log(
     `${chalk.cyan('> Success!')} Domain ${chalk.bold(
-      addedDomain.name
-    )} added correctly. ${addStamp()}\n`
+      domainName
+    )} added to project ${chalk.bold(projectName)}. ${addStamp()}`
   );
 
-  if (!addedDomain.verified) {
-    output.warn(
-      `The domain was added but it is not verified. To verify it, you should either:`
+  if (isPublicSuffix(domainName)) {
+    output.log(
+      `The domain will automatically get assigned to your latest production deployment.`
     );
-    output.print(
-      `  ${chalk.gray(
-        'a)'
-      )} Change your domain nameservers to the following intended set: ${chalk.gray(
-        '[recommended]'
-      )}\n`
+    return 0;
+  }
+
+  const domainResponse = await getDomain(client, contextName, domainName);
+
+  if (domainResponse instanceof Error) {
+    output.error(domainResponse.message);
+    return 1;
+  }
+
+  if (!domainResponse.verified) {
+    output.warn(
+      `The domain was added but it is not verified. ` +
+        `To verify it, you should change your domain nameservers to the following intended set`
```

Comment:
`verified` will always be `true` now.

We can replace this with GET `<v>/domains/:name/config` endpoint to check if the domain that was just added is configured properly